src/main/java/picard/illumina/parser/CbclData.java

```diff
@@ -42,4 +39,9 @@ public int getXCoordinate() {
     public int getYCoordinate() {
         return this.positionInfo.yQseqCoord;
     }
+
+    @Override
+    public boolean isPf() {
```

Comment:
In order to follow the usual conventions for java property getters/setters this should be `isPfRead()`